libs/packages/components/src/lib/search-result-list/search-result-list.component.ts

```diff
@@ -20,4 +28,34 @@ export class SdsSearchResultListComponent {
    * Child Template to be used to display the data for each item in the list of items
    */
   @ContentChild('resultContent') resultContentTemplate: TemplateRef<any>;
+
+  data: any = {
+    title: '',
+    message: ''
+  };
+
+  ngOnInit(){
+    this.displayTemplate();
+  }
+
+  ngOnChanges() {
+    this.displayTemplate();
+  }
+
+
+
+  displayTemplate() {
+    if (this.model.error) {
+      this.data.title = this.model.error.title;
+      this.data.message = this.model.error.description;
```

Comment:
error/emptySearch/noitems description property renamed message for readability

Reply:
I'm not sure that I understand the request here.

Reply:
changing description to message or message to description to keep property naming consistent